tableview.go

```diff
@@ -2,6 +2,8 @@ package clui
 
 import (
 	"fmt"
+	"strings"
```

Comment:
A general comment: could you add or change existing demo to show how to use such columns?

E.g, you can modify `demos/tableview.go` and unite two columns in the middle(I think, good candidates are "Number" and "Misc" - they are fairly small, adjacent and in the middle)

Reply:
Sure I will. I've been a bit busy, so I'll follow up on the next weekend.